mobile/src/libs/components/components.tsx

```diff
@@ -12,15 +12,18 @@ export { Link } from './link/link';
 export { Loader } from './loader/loader';
 export { Player } from './player/player';
 export { ProtectedRoute } from './protected-route/protected-route';
+export { SetTimerModal } from './set-timer-modal/set-timer-modal';
 export { SignBackground } from './sign-background/sign-background';
 export { Switch } from './switch/switch';
 export { Toast } from './toast/toast';
 export {
   Image,
   Pressable,
+  Modal as ReactModal,
```

Comment:
We have already created a custom Modal component. You can see it, for example, [here](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/pull/330)

Reply:
@danilittaa i think it should be separate modal espessially for timer. like in frontend we have different modal components

Reply:
You didn't understand me. Yes, of course, there should be a separate modal component, and it already exists (libs/components/set-timer-modal). But inside this component, we need to use our component from 'libs/components/modal'